cloudwatch_metrics_collector/package.xml

```diff
@@ -19,6 +19,7 @@
   <build_depend>rclcpp</build_depend>
   <build_depend>rmw_implementation</build_depend>
   <build_depend>ros_monitoring_msgs</build_depend>
+  <build_depend>metrics_statistics_msgs</build_depend>
```

Comment:
Note: is this package bloomed? If not, this PR will prevent cloudwatchmetrics from being bloomed again.

Reply:
Cloudwatch Metrics can't be bloomed again only until `metrics_statistics_msgs` is bloomed. They can even be bloomed at the same time.